input/intro-notes/StructureDefinition-au-core-condition-intro.md

```diff
@@ -25,7 +25,7 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 - `Condition.code` contains the identification of the condition, problem or diagnosis which may include body site information. Where a system has information not supported by the coding in `Condition.code.coding` (e.g. body site, laterality and other qualification of condition coding terms) that information **SHOULD** be supplied in `Condition.code.text`
 - An active condition is represented using "active" in `Condition.clinicalStatus`
 - To represent that a patient does not have a condition or history of condition, an appropriate negation code is used in `Condition.code`:
-  - For indicating no known conditions or problems of any type `Condition.code` **SHOULD** use the code `SNOMED CT: 160245001 | No current problems or disability (situation)`
+  - For indicating no known conditions or problems of any type `Condition.code` **SHOULD** use the SNOMED-CT 160245001 \|No current problems or disability (situation)\|
```

Comment:
It is "SNOMED CT"  not "SNOMED-CT"

Reply:
Fixed.

Reply:
noted fix. there is a grammar issue. restore the word "code" so it becomes "the code SNOMED CT"

Reply:
Done.